guides/common/modules/con_prerequisites-for-configuring-project-with-quarkus-keycloak-authentication.adoc

```diff
@@ -0,0 +1,23 @@
+[id="prerequisites-for-configuring-{project-context}-with-keycloak-authentication_{context}"]
+= Prerequisites for configuring {Project} with {Keycloak} authentication
+
+Before configuring {Project} with {Keycloak} external authentication, ensure that you meet the following requirements:
+
+* A working installation of {Keycloak} server that uses HTTPS instead of HTTP
+* A {Keycloak} account with admin privileges
+* A realm for {Project} user accounts created in {Keycloak}
+* If the certificates or the CA are self-signed, ensure that they are added to the end-user certificate trust store
+* Users imported or added to {Keycloak}
```

Comment:
```suggestion
* A working installation of {Keycloak} server that uses HTTPS instead of HTTP.
* A {Keycloak} account with admin privileges.
* A realm for {Project} user accounts created in {Keycloak}.
* If the certificates or the CA are self-signed, ensure that they are added to the end-user certificate trust store.
* Users imported or added to {Keycloak}.
```
Periods at the end of each item because line 9 is a complete sentence.